.circleci/config.yml

```diff
@@ -524,28 +524,36 @@ jobs:
           working_directory: ~/project/packages/storage
           command: cargo clippy --features iterator -- -D warnings
       - run:
-          name: Clippy linting on vm (no feature flags)
+          name: Clippy linting on vm (singlepass; no feature flags)
           working_directory: ~/project/packages/vm
           command: cargo +$NIGHTLY_TOOLCHAIN clippy -- -D warnings
       - run:
-          name: Clippy linting on vm (all feature flags)
+          name: Clippy linting on vm (singlepass; all feature flags)
           working_directory: ~/project/packages/vm
           command: cargo +$NIGHTLY_TOOLCHAIN clippy --features iterator -- -D warnings
+      - run:
```

Comment:
Always good to do. I like the improved caching. I was wondering why clippy was one of the slowest CI tasks.